sql-plugin/src/main/scala/com/nvidia/spark/rapids/ShimLoader.scala

```diff
@@ -209,24 +209,18 @@ object ShimLoader extends Logging {
   private def updateSparkClassLoader(): Unit = {
     // TODO propose a proper addClassPathURL API to Spark similar to addJar but
     //  accepting non-file-based URI
-    serializerClassloader().foreach { urlAddable =>
```

Comment:
This will break existing deployments for which this logic was critical. 

Yes it's not supported on vanilla JDK11 JVM. We can look at this deeper, but In the past experience `spark.rapids.force.caller.classloader=true` was really necessary only for advanced REPLs such as some multi-session notebook servers. We proposed it as the default value because it was safe on JDK8 and covered all the cases we have encountered. 

I would suggest to check if setting `spark.rapids.force.caller.classloader=false` works for your use cases https://github.com/NVIDIA/spark-rapids/issues/3851#issuecomment-992235387. 

If yes we should set it in code for JDK versions 11+.

Reply:
Thank you, will check it using workaround.

Reply:
Workaround works for me, will close this PR.